src/components/cart/Cart.tsx

```diff
@@ -0,0 +1,64 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import styles from './cart.module.scss';
+import { useState } from 'react';
+import store from '../../redux/store';
+
+function Cart() {
+  const result = store.getState().cart;
+  const { data } = result;
```

Comment:
```suggestion
  const { data } = result;
```
this line is unecessary